wordpress-mobile/WordPress-FluxC-Android
# Woo: Expose Shipping Label expiry date

This is needed to fix https://github.com/woocommerce/woocommerce-android/issues/3845, we need to disable reprinting expired shipping labels, and the check depends on the `expiry_date` field.
The PR adds also the following change: change the type of dates in `WCShippingLabelModel` to long, as storing them as TEXT doesn't add any value. 

#### Testing
1. Open the example app.
2. Click on Woo.
3. Click on Shipping Labels.
4. Click on Fetch Shipping Labels.
5. Confirm that the fetching works.

Please don't merge until the corresponding woo PR is reviewed.
